primefaces/src/main/java/org/primefaces/model/charts/axes/cartesian/CartesianScaleTitle.java

```diff
@@ -217,7 +237,8 @@ public String encode() throws IOException {
             fsw.write("{");
 
             ChartUtils.writeDataValue(fsw, "display", this.display, false);
-            ChartUtils.writeDataValue(fsw, "labelString", this.labelString, true);
+            ChartUtils.writeDataValue(fsw, "title", this.title, true);
```

Comment:
```suggestion
            ChartUtils.writeDataValue(fsw, "text", this.title, true);
```
https://www.chartjs.org/docs/latest/axes/labelling.html